src/bp-core/bp-core-functions.php

```diff
@@ -5286,3 +5286,14 @@ function bp_is_admin( $screen_id = '' ) {
 
 	return $screen_id === $bp->admin->current_screen;
 }
+
+/**
+ * Common function to check if bp_classic is active or not.
+ *
+ * @since 15.0.0
+ *
+ * @return bool True if bp_classic is active, false otherwise.
+ */
+function is_bp_classic_active() {
```

Comment:
One last suggestion to start with our custom prefix: `bp_`
```suggestion
function bp_is_classic() {
```

Reply:
@renatonascalves Thanks for the suggestion, I have implemented the same please check.

Reply:
@renatonascalves Please let me know if any other changes needs to be made.